pkg/oc/cli/admin/release/extract.go

```diff
@@ -46,6 +46,7 @@ func NewExtract(f kcmdutil.Factory, parentName string, streams genericclioptions
 		},
 	}
 	flags := cmd.Flags()
+	flags.StringVarP(&o.AuthConfig, "auth-config", "a", o.AuthConfig, "Path to your registry credentials (defaults to ~/.docker/config.json)")
```

Comment:
`auth-config` should be for OpenShift auth related stuff, so I'm proposing this to be `image-config` or `registry-config` or anything that will clearly show it's for image registry not auth.

Reply:
Registry config is fine.